pkg/ovs/daemonset.go

```diff
@@ -111,16 +108,10 @@ func DaemonSet(
 						{
 							Name: ServiceName + "db-server",
 							Command: []string{
-								"/usr/local/bin/container-scripts/start-ovsdb-server.sh",
+								"/usr/bin/start-ovs",
```

Comment:
where do you get it from? the official image we use for the container - openstack-ovn-base - doesn't contain the script. Hence the need to drive the startup of services via a shell script.

Reply:
I did not plan to update this, must be a bad rebase. will fix it